mods/ts/rules/nod-support.yaml

```diff
@@ -296,6 +296,9 @@ NAMISL:
 		DisplayBeacon: False
 		DisplayRadarPing: True
 		BeaconPoster:
+		ClockSprite:
```

Comment:
Nulling out properties in yaml isn't very nice.  The default should be empty, and the other mods override it with their additions.